react/OrderPayment.tsx

```diff
@@ -140,52 +117,41 @@ export const OrderPaymentProvider: React.FC<OrderPaymentProps> = ({
     [enqueue, queueStatusRef, setOrderForm, updateOrderFormPayment]
   )
 
-  const savePaymentData = useCallback(
-    async (paymentData: PaymentData[]): Promise<Status> => {
-      try {
-        const {
-          data: { saveCards },
-        } = await saveCardsMutation({
-          variables: {
-            paymentData,
-            cardSessionId: cardSession.getCardSessionId,
-          },
-        })
-
-        const { tokenizedCards } = saveCards
-
-        await savePaymentTokenMutation({
-          variables: {
-            paymentTokens: getPaymentTokens(tokenizedCards),
-          },
-        })
-
-        return {
-          error: false,
-          value: tokenizedCards,
-        }
-      } catch (err) {
-        refreshCardSession()
-
-        return {
-          error: true,
-          value: 'apiError',
-        }
+  const setPaymentField = useCallback(
+    async (paymentField: Partial<PaymentInput>) => {
```

Comment:
you don't need the `async` keyword here
```suggestion
    (paymentField: Partial<PaymentInput>) => {
```